add-chain/flags/flags.go

```diff
@@ -0,0 +1,64 @@
+package flags
+
+import (
+	"github.com/urfave/cli/v2"
+)
+
+const EnvVarPrefix = "SCR"
+
+func prefixEnvVars(names ...string) []string {
```

Comment:
Curious why we accept a variadic slice of strings but only ever pass one string in? Is this because this is how it is done in the monorepo?

Reply:
Yes just used the same helper function as in op-node